### PR TITLE
Display Winner with Text box appearing on screen

### PR DESCRIPTION
**Pull request by farhaannishtar**
I'm able to display the winner ASAP with a popup of a text box.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -21,19 +21,30 @@ function App() {
...
-    console.log(calculateWinner(cloneBoardValues) + ' is the winner');
+    let winner = calculateWinner(cloneBoardValues);
+    console.log(winner);
+    const gameOver = cloneBoardValues.every((element: string) => element !== '-');
+    if (winner !== '-') {
+      displayWinner(winner);
+    } else if (gameOver && winner === null) {
+      displayWinner(winner);
+    }
+  }
```
Comment (ahsojar) [inline src/App.tsx:39]: This code can be deleted--it's not actually doing anything.

Your `displayWinner` returns a string, and so the way you're using it in line 69 is correct-- rendering it in the React tsx code. 

Also, we already define const `winner` in line 14--- so we don't need to redefine it on line 31.

File: src/components/helpers.ts [hunk 2]
```diff
@@ -41,5 +41,8 @@ export function calculateWinner(squares: string[]) : string | null {
   if (squares[2] === squares[4] && squares[4] === squares[6]) {
     return squares[2];
   }
-  return null
+  if (squares.every((element: string) => element !== '-')) {
+    return null;
```
Comment (ahsojar) [inline src/components/helpers.ts:45]: Is this the tie case? If so we might want to just return 'tie' to make it extremely clear. 

As someone reading this code, it isn't super obvious the difference between returning `null` and `''`. Having it return `tie` will just make the logic easier to follow. What do you think?

Comment (farhaannishtar) [reply src/components/helpers.ts:45]: This is actually the case where if all the squares don't equal to '-', then return null.

Every square starts out as '-', so until all squares are clicked, null is returned.

Comment (ahsojar) [reply src/components/helpers.ts:45]: Makes sense-- we're checking this to basically see if it's a `tie` though, right? AKA, the entire board is filled out, but there is no winner

**Pull request merged**